pkg/cmd/stack/translate.go

```diff
@@ -158,7 +158,7 @@ func translateBuildImages(ctx context.Context, s *model.Stack, options *StackDep
 		}
 
 		volumesToInclude := svcInfo.VolumeMounts
-		var options *build.BuildOptions
+		options := &build.BuildOptions{}
```

Comment:
is there any other definition like `var options *build.BuildOptions` we should replace it everywhere

Reply:
i havent seen more